examples/two-step-transfer/Cargo.toml

```diff
@@ -24,6 +22,11 @@ version = "4"
 path = "../../concordium-std"
 default-features = false
 
+[dependencies.concordium-contracts-common]
```

Comment:
We don't want contracts to depend on `concordium-contracts-common`.
`concordium-std` reexports `concordium-contracts-common` so it should be used through there.

Reply:
Fixed.